README.md

```diff
@@ -967,6 +967,7 @@ be configured by setting `LEVEL_TYPE` to an expected type, such as
 - AMPLIFIED
 - CUSTOMIZED
 - BUFFET
+- BIOMESOP (if Biomes O' Plenty is installed)
```

Comment:
Since this list only serves as examples now, you could clarify the `BIOMESOPLENTY` you found for 1.15+